sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -100,6 +100,17 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
       }
     }
   }
+
+  test("no database specified") {
+    Seq(
+      s"SHOW TABLES IN $catalog",
```

Comment:
This deserves more discussion. For `SELECT * FROM catalog`, we fill the current catalog and namespace for this single-part table name.

For `SHOW TABLES IN catalog`, we are resolving namespaces, and I think we should fill the current catalog for this single-part namespace name.